src/Service.Host/client/src/components/Pallets.js

```diff
@@ -0,0 +1,596 @@
+                        <Grid container spacing={2}>
+                            <Grid size={2}>
+                                <Dropdown
+                                    value={pallet?.accessible}
+                                    fullWidth
+                                    label="Acessible"
+                                    propertyName="accessible"
+                                    allowNoValue
+                                    items={['Y', 'N']}
+                                    onChange={handleFieldChange}
+                                />
+                            </Grid>
+                            <Grid size={2}>
+                                <DatePicker
+                                    label="Date Invalid"
+                                    propertyName="dateInvalid"
+                                    value={pallet?.dateInvalid}
+                                    onChange={date => handleDateChange('dateInvalid', date)}
+                                    clearable
+                                />
+                            </Grid>
+                            <Grid size={2}>
+                                <Dropdown
+                                    value={pallet?.cage}
+                                    fullWidth
+                                    label="Cage"
+                                    propertyName="cage"
+                                    allowNoValue
+                                    items={['Y', 'N']}
+                                    onChange={handleFieldChange}
+                                />
+                            </Grid>
+                        </Grid>
+                    </Grid>
+                    <Grid size={12}>
+                        <Grid container spacing={2}>
+                            <Grid size={2}>
+                                <Dropdown
+                                    value={pallet?.storesKittable}
+                                    fullWidth
+                                    label="Stores Kittable"
+                                    propertyName="storesKittable"
+                                    allowNoValue
+                                    items={['Y', 'N']}
+                                    onChange={handleFieldChange}
+                                />
+                            </Grid>
+                            <Grid item size={4}>
+                                <InputField
+                                    propertyName="storesKittingPriority"
+                                    label="Stores Kitting Priority"
+                                    value={pallet?.storesKittingPriority}
+                                    fullWidth
+                                    onChange={handleFieldChange}
+                                    type="number"
+                                />
+                            </Grid>
+                            <Grid size={2}>
```

Comment:
you are being quite inconsisten with whether or not you pass these Grid components the 'item' prop. IIRC we don't need it at all anymore